style.css

```diff
@@ -368,110 +376,187 @@ li:hover {
   display: flex;
   align-items: center;
   color: #000;
-  padding-left: 30px;
 }
 
-#sk {
-  left: 12px;
-  right: 78px;
+#JAVASCRIPT {
+  width: 303px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 20px;
+  content: "text";
   font-family: 'Poppins', 'Times New Roman', Times, serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px;
+  display: flex;
   align-items: center;
-  color: #000;
-  padding-left: 30px;
-  display: inline-block;
+  color: #253858;
 }
 
-.fa-angle-right {
-  padding-left: 280px;
+#HTML {
+  width: 303px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  flex: none;
+  padding: 10px;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 20px;
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
 }
 
-.il {
-  display: inline;
+#CSS {
+  width: 303px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
   padding: 10px;
+  display: flex;
+  align-items: center;
+  color: #253858;
 }
 
-.fs {
-  display: inline;
-  width: 100px;
-  padding-left: -20px;
+.Pointer-1 {
+  padding-left: 150px;
 }
 
-.fa-angle-down {
-  padding-left: 110px;
-} */
+.Pointer-2 {
+  padding-left: 147px;
+}
 
-#la {
-  margin-top: 30px;
-  margin-bottom: 30px;
-  height: 24px;
+.Pointer-3 {
+  padding-left: 220px;
+}
+
+.Contact-me {
+  width: 375px;
+  height: 750px;
+  border-radius: 10px;
+  padding-bottom: 20px;
+  background: #6070ff;
+  background-image: url("images/contactbg.png");
+  background-repeat: no-repeat;
+  background-position: 180px 20px;
+  margin-top: -98px;
+  visibility: visible;
+  background-size: 200px;
+  border-top-left-radius: 50px;
+}
```

Comment:
- According to the project requirements, please use the `flexbox` display property to align items in this section. This allows for more flexibility and responsiveness 🙏